### PR TITLE
docs(usage): add documentation for Prowler App usage

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Add documentation about the usage of the new Prowler App:

- Explain how to interact with the app.
- Provide a brief walkthrough with screenshots of the app's user interface.

### Checklist

- Are there new checks included in this PR? Yes / No
    - If so, do we need to update permissions for the provider? Please review this carefully.
- [ ] Review if the code is being covered by tests.
- [ ] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [ ] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/tutorials/prowler-app.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    <img src="../../img/aws-credentials.png" alt="AWS Credentials" width="350"/>
+
+#### **Step 4.1.2: IAM Role**
+1. Select `Connect assuming IAM Role`.
+
+    <img src="../../img/connect-aws-role.png" alt="AWS Role" width="350"/>
+
+2. Enter the `Role ARN` and any optional field like the AWS Access Keys to assume the role, the `External ID`, the `Role Session Name` or the `Session Duration`:
+
+    <img src="../../img/aws-role.png" alt="AWS Role" width="350"/>
```
Comment (puchy22) [inline docs/tutorials/prowler-app.md:65]: I think this image should be larger, I can barely see anything even if I zoom in. I have to open it in another window.
At least on my monitor it has room to have a bigger width.

File: docs/tutorials/prowler-app.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+---
+
+### **Step 4.2: Azure Credentials**
```
Comment (puchy22) [inline docs/tutorials/prowler-app.md:69]: I miss at least one image of the interface for the rest of providers, it is not mandatory change but it is an observation

File: docs/tutorials/prowler-app.md (hunk 1 above)
Comment (puchy22) [inline docs/tutorials/prowler-app.md]: I think this image should be larger, I can barely see anything even if I zoom in. I have to open it in another window.
At least on my monitor it has space to have a bigger width.
```suggestion
    <img src="../../img/aws-role.png" alt="AWS Role" width="700"/>
```

File: docs/tutorials/prowler-app.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+For Kubernetes, Prowler App uses a `kubeconfig` file to authenticate, paste the contents of your `kubeconfig` file into the `Kubeconfig content` field.
+
+By default, the `kubeconfig` file is located at `~/.kube/config`.
+
+---
+
+### **Step 5: Test Connection**
+After adding your credentials of your cloud account, click the `Test Connection` button to verify that the Prowler App can successfully connect to your provider:
+
+<img src="../../img/test-connection-button.png" alt="Test Connection" width="700"/>
```
Comment (puchy22) [inline docs/tutorials/prowler-app.md:100]: Minor note, the mouse has slipped into the lower right corner of the photo. I don't think it's bad but just to keep it in mind.

File: docs/tutorials/prowler-app.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+<img src="../../img/scan-progress.png" alt="Scan Progress" width="700"/>
+
+---
+
+## **Step 8: Analyze the Findings**
+While the scan is running, start exploring the findings in these sections:
+
+- **Overview**: High-level summary of the scan.
+- **Findings**: Detailed list of issues detected. <img src="../../img/findings.png" alt="Findings" width="700"/>
```
Comment (puchy22) [inline docs/tutorials/prowler-app.md:123]: In the first column appears checks when it should be findings/check description, it is not a problem of the documentation but of the UI but it should be changed in the future with a photo that fixes that.

File: docs/tutorials/prowler-app.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+- **Compliance**: Insights into compliance status. <img src="../../img/compliance.png" alt="Compliance" width="700"/>
+- **Services**: Breakdown of services scanned. <img src="../../img/services.png" alt="Services" width="700"/>
+
+???+ note
+    Prowler will automatically scan all configured providers every **24 hours**, ensuring your cloud environment stays continuously monitored.
```
Comment (puchy22) [inline docs/tutorials/prowler-app.md]: I think this note should be in step 6 since this is when the scanning starts every 24 hours.

**Review by puchy22 — CHANGES_REQUESTED** [review]

File: docs/tutorials/prowler-app.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+After adding your credentials of your cloud account, click the `Test Connection` button to verify that the Prowler App can successfully connect to your provider:
+
+<img src="../../img/test-connection-button.png" alt="Test Connection" width="700"/>
+
+
+## **Step 6: Start Your First Scan**
+After successfully adding and testing your credentials, click the `Start Now` button to initiate your first scan:
+
+<img src="../../img/start-now.png" alt="Start Now" width="700"/>
+
```
Comment (puchy22) [inline docs/tutorials/prowler-app.md:107]:
```suggestion

???+ note
    Prowler will automatically scan all configured providers every **24 hours**, ensuring your cloud environment stays continuously monitored.
```

File: docs/tutorials/prowler-app.md (hunk 5 above)
Comment (puchy22) [inline docs/tutorials/prowler-app.md]:
```suggestion
```

File: docs/tutorials/prowler-app.md (hunk 4 above)
Comment (MrCloudSec) [reply docs/tutorials/prowler-app.md:123]: I will change it whenever this is changed, but good catch!

File: docs/tutorials/prowler-app.md (hunk 3 above)
Comment (MrCloudSec) [reply docs/tutorials/prowler-app.md:100]: Done!

File: docs/tutorials/prowler-app.md (hunk 2 above)
Comment (MrCloudSec) [reply docs/tutorials/prowler-app.md:69]: Done!

File: docs/index.md [hunk 7]
```diff
@@ -231,6 +231,42 @@ We have deprecated some of our outputs formats:
 - To send only FAILS to AWS Security Hub, now use either `--send-sh-only-fails` or `--security-hub --status FAIL`.
 
 ## Basic Usage
+### Prowler App
+
+#### **Access the App**
+Go to [http://localhost:3000](http://localhost:3000) after installing the app (see [Quick Start](#prowler-app-installation)). Sign up with your email and password.
```
Comment (puchy22) [inline docs/index.md:237]: This is not working but I think because it should be from other PR so I did not count this as an error

**Review by puchy22 — DISMISSED** [review]
Great job @MrCloudSec 🥇

**Review by puchy22 — DISMISSED** [review]
👍

**Review by pedrooot — DISMISSED** [review]
🔝 🔝 🔝 🔝

**Review by pedrooot — APPROVED** [review]
🔝

**Review by puchy22 — APPROVED** [review]
🥷

**Pull request merged**